src/js/gameController.js

```diff
@@ -198,18 +199,23 @@ var gameController = {
         this.player.anchor.setTo(0.5,1.0);
         game.physics.arcade.enable(this.player, Phaser.Physics.ARCADE);
         this.player.body.kinematic = true;
-        this.player.inputEnabled = true;
-        this.player.input.enableDrag(false, true, true);
-        this.player.input.allowVerticalDrag = false;
         this.player.collideWorldBounds = true;
-        let bounds = new Phaser.Rectangle(0,0, canvasWidth, canvasHeight);
-        this.player.input.boundsRect = bounds;
```

Comment:
This entails the basket moving off the screen.
We should prevent this from happening. Reverting the 2 lines above should solve the issue.